bin/pygrb/pycbc_grb_trig_combiner

```diff
@@ -486,7 +454,8 @@ logging.info(
 # there should always be two segments in the offsource bins,
 # one before and one after the onsource.
 # Nevertheless, simply rely on there being at least one stretch of offsource.
-offsource_ntrials = [int(numpy.trunc(abs(b)/trialtime)) for b in bins["OFFSOURCE"]]
+offsource_ntrials = [int(numpy.trunc(abs(b)/trialtime))
+                     for b in bins["OFFSOURCE"]]
```

Comment:
I am pretty sure this produces the same result given that all quantities are positive:
```suggestion
offsource_ntrials = [int(abs(b) / trialtime) for b in bins["OFFSOURCE"]]
```

Reply:
True